metadata/metadata.go

```diff
@@ -98,11 +99,43 @@ type RealUpdater struct {
 	Local bool
 }
 
+// findIP extracts an ip address from a string using a regex pattern.
+func findIP(input string) string {
+	if input == "" {
+		return input
+	}
+	numBlock := "(25[0-5]|2[0-4][0-9]|1[0-9][0-9]|[1-9]?[0-9])"
+	regexPattern := numBlock + "\\." + numBlock + "\\." + numBlock + "\\." + numBlock
+
+	regEx := regexp.MustCompile(regexPattern)
+	return regEx.FindString(input)
+}
+
 func (r RealUpdater) getAddress() string {
-	if r.Local {
+	if !r.Local {
```

Comment:
what's wrong with allowing people to use a hostname for their DOCKER_HOST?

Reply:
!r.Local means the build happens inside the Cloud Build service. Perhaps a poorly chosen name, since people could run their own remote version of this binary and want to connect, but there is no choice to be had in the hosted version.